dbms/src/Storages/DeltaMerge/Index/InvertedIndex/Reader/ColumnFileTiny.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Common/Stopwatch.h>
+#include <Common/TiFlashMetrics.h>
+#include <IO/Buffer/ReadBufferFromString.h>
+#include <IO/Compression/CompressedReadBuffer.h>
+#include <Storages/DeltaMerge/ColumnFile/ColumnFileDataProvider.h>
+#include <Storages/DeltaMerge/ColumnFile/ColumnFileTiny.h>
+#include <Storages/DeltaMerge/Index/InvertedIndex/Reader.h>
+#include <Storages/DeltaMerge/Index/InvertedIndex/Reader/ColumnFileTiny.h>
+#include <Storages/DeltaMerge/Index/LocalIndexCache.h>
+
+namespace DB::DM
+{
+
+InvertedIndexReaderFromColumnFileTiny::InvertedIndexReaderFromColumnFileTiny(
+    const ColumnRangePtr & column_range_,
+    const ColumnFileTiny & tiny_file_,
+    const IColumnFileDataProviderPtr & data_provider_,
+    const LocalIndexCachePtr & local_index_cache_)
+    : tiny_file(tiny_file_)
+    , data_provider(data_provider_)
+    , column_range(column_range_)
+    , local_index_cache(local_index_cache_)
+{
+    GET_METRIC(tiflash_inverted_index_active_instances, type_memory_reader).Increment();
+}
+
+BitmapFilterPtr InvertedIndexReaderFromColumnFileTiny::load()
+{
+    if (loaded)
+        return nullptr;
```

Comment:
Calling `load` more than once will return `nullptr`. Is it expected that all these reader will call `load` only once?

Reply:
reader can not be reused, so can only call `load` once.